pkg/util/oc/oc_struct.go

```diff
@@ -219,6 +235,14 @@ func (o OC) GetConfigMapData(t test.TestHelper, ns, name string) map[string]stri
 	return data
 }
 
+func (o OC) CreateConfigMapFromFile(t test.TestHelper, ns, name, file string) {
+	t.T().Helper()
+	o.withKubeconfig(t, func() {
+		t.T().Helper()
+		o.Invokef(t, "oc create configmap -n %s %s --from-file=%s", ns, name, file)
+	})
+}
```

Comment:
This also duplicates the code in `CreateSecretFromFile`, so a better approach would be to refactor that function into a `CreateSecretOrConfigMapFromFile(t, ns, kind, name, file)` and have both functions call it.

Reply:
Done, but check if is the best way to do it.

Reply:
Yeah, I didn't mean the tests to call `CreateSecretOrConfigMapFromFile` directly, as that makes the tests less simple (since you have to specify the kind parameter). I said "... and have both functions call it". So, I wanted you to keep the existing functions `CreateGenericSecret` and `CreateConfigMap`, but they should call the `CreateSecretOrConfigMapFromFile`. 

Anyway, I fixed the code myself.